src/Posts/Post.jsx

```diff
@@ -248,15 +286,44 @@ return (
     )}
     {!state.hasBeenFlagged && (
       <Post>
+        <>
+          {repostData ? (
+            <Widget
+              src="${REPL_ACCOUNT}/widget/AccountProfileOverlay"
+              props={{
+                accountId,
+                profile,
+                children: (
+                  <div className="row">
+                    <div className="col-auto ">
+                      <i className="bi bi-repeat" /> reposted by{" "}
+                      <TextLink href={repostedByProfileUrl} ellipsis>
+                        @{repostData.reposted_by}
+                      </TextLink>
```

Comment:
This comment related to the comment above:

```suggestion
                      <TextLink href={repostedByProfileUrl} $ellipsis>
                        @{repostData.reposted_by}
                      </TextLink>
```

Other way, there will be an error in the console about unsupported property `ellipsis` for `<a>` tag

Reply:
has this been resolved?